src/nile/utils/status.py

```diff
@@ -0,0 +1,100 @@
+"""Functions used to find/track/debug a transaction status."""
```

Comment:
For me, this file looks weird under the utils folder. I think we should check the issue for organizing the code structure soon because is going to be harder while the codebase continues growing.

This would loog better in "core/commands/helpers" IMO.